Packs/Synapse/Integrations/Synapse/Synapse.py

```diff
@@ -417,34 +409,26 @@ def ip_reputation_command(client, args, good_tag, bad_tag):
             dbot_score=dbot_score
         )
 
-        ip_standard_list.append(ip_standard_context)
-        ip_data_list.append(ip_data)
-
-    readable_output = tableToMarkdown('IP List', ip_data_list)
+        command_results.append(CommandResults(
+            readable_output=tableToMarkdown('IP List', ip_data),
```

Comment:
Given a single IP, I feel that `IP List` is not a good human-readable title, perhaps `IP Details`?